Antidote/DialingCallViewController.m

```diff
@@ -84,8 +106,11 @@ - (void)updateFriendAvatar
 {
     AvatarsManager *avatars = [AppContext sharedContext].avatars;
 
+    CGFloat smallestSide = MIN(self.view.bounds.size.width, self.view.bounds.size.height);
+    CGFloat diameterOfImage = smallestSide / 2;
```

Comment:
Hm... this value will always be the same. It doesn't matter how you rotate device - MIN(width, height) or MIN(height, width) will always give you same value :wink: